### DISCUSSION TITLE
Does concourse have authentication methods for automation? Like service accounts or certificate authorization

### DISCUSSION THREAD

**Discussion by parunp** [concourse/concourse#8924] [Help & Support] [answered]
I am looking for assistance related to Concourse authentication/authorization methods. 
We're working on an automation that requires us to trigger a Concourse job from a Jenkins pipeline and we have an issue with authentication to the Concourse team.
We're using github OAuth as a provider, so concourse login will trigger an interactive session for github authentication and store the token. We won't be able to use local user auth for security compliance.

Please let me know if there is an alternate method that can eliminate the interactive session.

**Comment by taylorsilva** [comment, marked as answer]
There isn't a way to do this. Folks usually trigger a pipeline through a resource. If you can't find a resource-type that fits your use-case then it's recommended to make your own resource-type.

If you want to be really hacky you can try and do something on top of the local auth method: https://concourse-ci.org/local-auth.html
